spec/views/groups/show.html.erb_spec.rb

```diff
@@ -7,14 +7,20 @@
   let(:upgrade?) { false }
   let(:edit?) { true }
   let(:request_upgrade?) { false }
+  let(:review_upgrade?) { false }
 
   before do
     assign(:current_user, current_user)
     assign(:group, group)
     assign(:forms, forms)
 
     without_partial_double_verification do
-      allow(view).to receive(:policy).and_return(instance_double(GroupPolicy, upgrade?: upgrade?, edit?: edit?, request_upgrade?: request_upgrade?))
+      double = instance_double(GroupPolicy,
+                               upgrade?: upgrade?,
+                               edit?: edit?,
+                               request_upgrade?: request_upgrade?,
+                               review_upgrade?: review_upgrade?)
```

Comment:
Does the shorthand work here?
```suggestion
                               upgrade?:,
                               edit?:,
                               request_upgrade?:,
                               review_upgrade?:)
```

Reply:
Unfortunately not in this case, I think it's because of the question marks 🤔